docs/studio/Dashboards.md

```diff
@@ -0,0 +1,63 @@
+# Dashboards
+
+## What is a dashboard?
+
+A `Dashboard` is simply a query with a description and label. This will be used by the `Workspace` to query against the view paired with it inside the `Workspace` configuration. This way, you can write one query, and re-use it against multiple potential views.
+
+An example `Dashbaord` resource looks like this:
+
+```json
+{
+  "@context": "https://bluebrainnexus.io/studio/context",
+  "@type": "StudioDashboard",
+  "dataQuery": "prefix nxv: <https://bluebrain.github.io/nexus/vocabulary/>\nprefix s: <http://schema.org/>\nSELECT DISTINCT ?self ?familyName ?givenName\nWHERE {\n?s nxv:constrainedBy <https://neuroshapes.org/dash/person> ;\n  nxv:self ?self ;\n  s:familyName ?familyName ; \n  s:givenName ?givenName\n}\nLIMIT 20",
+  "description": "A list of every scientist",
+  "label": "Scientists",
+  "plugins": ["nexus-plugin-example"]
+}
+```
+
+The `Dashboard`, once configured, will present the results of your sparql query in a table. Each item in the row will be clickable and lead to a details page showing the `Resource`, in this case, a list of scientists that have contributed data to the project.
+
+![Create a workspace form](../assets/dashboards-list.png)
+
+## Plugins (Experimental!)
+
+There's also an experimantal `plugin` key in the configuration that can be used to load experimental nexus plugins. These plugins will be loaded into the `Details` page of a resource.
+
+> [plugin development documentation](./docs/pluginDevelopment.md).
+
+## Creating a dashboard?
+
+Inside any `Studio View`, under a selected `Workspace`, click the `Add Dashboard` button.
+
+![Add a workspace](../assets/add-dashboard-button.png)
+
+..and fill in a form providing the following:
+
+- `Label`: the name of your new `Dashboard` (required field)
+- `Description`: an optional description for your `Dashboard`
+- `Plugins`: an optional list of plugins to add to the details page of a query result
+- `Sparql Query`: a valid [SPARQL](https://www.w3.org/TR/rdf-sparql-query/)query that conforms to the `dataQuery` field in the configuration.
```

Comment:
The sparql query requires specific headers to be returned in order for studio to work properly such as:
- `?self` for linking to the details page
- `?total` which is a count of the total elements in order to get the pagination working

These should be documented as well and maybe a base query template could be documented too?

Reply:
I've added documentation for the ?self header, but ?total is not required